airbyte-integrations/connectors/source-sendgrid/integration_tests/abnormal_state.json

```diff
@@ -42,5 +42,16 @@
         "name": "invalid_emails"
       }
     }
+  },
+  {
```

Comment:
Are we adding incremental sync type for 'suppression_group_members'? 
If yes, we need to mention this in the documentation under the streams list.

Reply:
Looks like this stream is now incremental - added a note to the `Supported Streams` list